packages/twig-new/vite.config.ts

```diff
@@ -0,0 +1,135 @@
+import chalk from "chalk";
+
+import theme from "@ilo-org/themes/tokens/theme/base.json" assert { type: "json" };
+
+const behaviors = globSync("src/components/**/*.behavior.js");
+
+export default defineConfig({
+  plugins: [
+    viteStaticCopy({
+      targets: [
+        {
+          src: "src/components/**/**.component.yml",
+          dest: "components/",
+          rename: (name, extension, fullPath) => {
+            const componentFolder = fullPath.match(
+              /(.*)\/(.*)\/(.*).component.yml$/
+            )?.[2];
+            return `${componentFolder}/${name}.${extension}`;
+          },
+        },
+        {
+          src: "src/components/**/**.twig",
+          dest: "components/",
+          rename: (name, extension, fullPath) => {
+            const componentFolder = fullPath.match(
+              /(.*)\/(.*)\/(.*).twig$/
+            )?.[2];
+            return `${componentFolder}/${name}.${extension}`;
+          },
+          transform: (contents, path) =>
+            contents
+              .toString()
+              .replace(/{{prefix}}/g, theme.themeprefix.value)
+              .replace(/prefix ~/g, `'${theme.themeprefix.value}' ~`),
+        },
+        {
+          src: "node_modules/@ilo-org/styles/css/index.css",
+          dest: "global/",
+          rename: "styles.css",
+        },
+      ],
+    }),
+    {
+      name: "post-build",
+      buildStart() {
+        console.log(
+          `${chalk.green("⚙︎")} ${chalk.yellow("rollup:")} ${chalk.green(
+            "building for production... \n"
+          )}`
+        );
+      },
+      buildEnd() {
+        const grey = chalk.hex("#A5ABB6");
+        console.log(
+          `${chalk.green("✄")} ${chalk.blue("vite:")} ${grey("copied: ")}`
+        );
+        console.log(chalk.green("  - .twig files"));
+        console.log(chalk.green("  - .component.yml files"));
+        console.log(chalk.green("  - global styles\n"));
+
+        for (const behavior of behaviors) {
+          build(behavior);
+        }
+      },
+    },
+  ],
+  logLevel: "error",
+  build: {
+    copyPublicDir: false,
+    rollupOptions: config(behaviors.at(0) as string),
+    chunkSizeWarningLimit: 800,
+  },
+});
+
+function config(id: string): NonNullable<BuildOptions["rollupOptions"]> {
+  return {
+    input: id,
+    output: {
+      dir: "dist",
+      format: "umd" as const,
+      entryFileNames(chunkInfo) {
+        const componentFolder = chunkInfo.facadeModuleId?.match(
+          /(.*)\/(.*)\/(.*).behavior.js$/
+        )?.[2];
+
+        return `components/${componentFolder}/${chunkInfo.name}.js`;
+      },
+    },
+  };
+}
+
+async function build(id: string) {
+  const options = config(id);
+  const startTime = Date.now();
+  const inputOptions: RollupOptions = {
+    input: options.input,
+    plugins: [
+      nodeResolve(),
+      commonjs(),
+      swc({
+        swc: {
+          env: {
+            targets: "last 2 versions, not dead, > 1% in US, ie 11",
```

Comment:
I think we can drop IE11, what do you think @justintemps?

Reply:
I can agree if someone wants to support `IE 11` they can re-bundle during their build step

Reply:
@bashlk and @GGKapanadze Yea we can definitely drop support for IE11